hack/lib/version.sh

```diff
@@ -40,8 +41,8 @@ kube::version::get_version_vars() {
   # we likely don't have a git tree, but these magic values may be filled in.
   if [[ '$Format:%%$' == "%" ]]; then
     KUBE_GIT_COMMIT='$Format:%H$'
-    KUBE_GIT_TREE_STATE="git archive"
-    # When a 'git archive' is exported, the '$Format:%D$' below will look
+    KUBE_GIT_TREE_STATE="git-archive"
```

Comment:
even simpler option: maybe remove the `git `, leaving this as `archive`? the "git" part is pretty redundant, anyway. `GitTreeState:"archive"` seems clear to me.